aii-dhcp/src/main/pan/quattor/aii/dhcp/config.pan

```diff
@@ -2,24 +2,31 @@
 # ${developer-info}
 # ${author-info}
 # ${build-info}
-template quattor/aii/${project.artifactId}/config;
+template quattor/aii/dhcp/config;
 
-include 'quattor/aii/${project.artifactId}/schema';
+include 'quattor/aii/dhcp/schema';
 
-# TFTP server
-# This is optional. Only it is necessary if your TFTP server is running on
-# a different machine than the DHCP server
-#
-# "/system/aii/dhcp/options/tftpserver" = "tftp.mydomain.org"
-#
+bind "/system/aii/discovery/dhcp" = structure_dhcp_module_info;
+
+prefix "/system/aii/discovery/dhcp";
+
+@documentation{
+    Enable the plugin
+}
+"enabled" = true;
+
+bind "/system/aii/dhcp" = structure_dhcp_dhcp_info;
+
+prefix "/system/aii/dhcp";
+
+@documentation{
+    Override the TFT server for this node
+}
 variable AII_DHCP_TFTPSERVER ?= null;
-"/system/aii/dhcp/options/tftpserver" ?= AII_DHCP_TFTPSERVER;
-
-# Additional DHCP options (optional).
-# Warning: They will be added in the host declaration of dhcpd.conf file, so
-# do not forget the ';' at the end
-#
-#"/system/aii/dhcp/addoptions" = "options blu-blo-bli bla;";
-#
+"tftpserver" ?= AII_DHCP_TFTPSERVER;
+
+@documentation{
+    Additional options to include in the host definition
+}
 variable AII_DHCP_ADDOPTIONS ?= null;
-"/system/aii/dhcp/options/addoptions" ?= AII_DHCP_ADDOPTIONS;
+"options" ?= AII_DHCP_ADDOPTIONS;
```

Comment:
shoudl be `"options/addoptions"`

Reply:
No, it should not :-) The original `aii-dhcp` command had a command-line option called `--addoptions` - which, due to being a command line option, needed to be a single string. But the plugin version never had "addoptions".

In other words - originally, the DHCP plugin did not come with templates, which was unfortunate. When templates were eventually added, they did not match what the code did. This change makes the templates match the code.

Reply:
we use the `options/addtoptions` value; at the very least this change is backwards incompatible
same for the relocation of the tftpserver from `options/tftpserver` to simple `tftpserver`.
in the `Shellfe.pm` is a methods called `dhcp` that uses this. please also fix that code if you plan to cleanup the schema.

Reply:
No, it is not backwards incompatible - _using_ addoptions in the plugin's templates would be backwards incompatible :-)

The underlying issue is, `aii-core/src/main/perl/DHCP.pm` is not compatible with the DHCP plugin - and therefore, you cannot use the the same templates. Which is unfortunate, but this is the case ever since the plugin exists. `aii-dhcp/src/main/pan/quattor` contains templates for the plugin - so it must be compatible with what the plugin is doing.

This PR does _not_ try to address the incompatibilities between the two implementations. But I think it is successfully pointing out where the incompatibilities are :-)